sites/friday/src/increase-observability/application-metrics/blackfire.md

```diff
@@ -0,0 +1,181 @@
+On this Blackfire environment, you have access to [all the features provided by Blackfire](https://www.blackfire.io/features/).
+This includes monitoring, profiling, alerting, and build-related features.
+
+Note that Blackfire monitoring is enabled by default on your production environment.
+On other environment types, you need to [enable it](#blackfire-monitoring).
+User access settings are replicated from the {{< vendor/name >}} Console to Blackfire.
+
+This includes all [access levels](https://blackfire.io/docs/up-and-running/access-management).
+ 
+You might have Blackfire variables already set on your project.
+In this case, the existing variables override the settings of the automated integration.
+
+{{< note >}}
+To trigger the synchronization of changes to users and their access levels,
+you need to redeploy the environment.
+{{< /note >}}
+
+### Blackfire monitoring
+
+Blackfire monitoring is enabled by default on your production environment.
+To enable Blackfire monitoring on your development or staging environments, follow these steps:
+
+1.  Go to your [organizations list](https://blackfire.io/my/organizations)
+    and select the organization where you want to enable Blackfire monitoring.
+
+2.  Click **Organization Monitoring Usage**.
+
+    ![A screenshot of where to find Organization Monitoring Usage](/images/integrations/blackfire/blackfire-organization-monitoring.png "0.40")
+
+3.  In the **Monitoring Activation** section,
+    enable monitoring on the environments of your choice.
+
+    ![A screenshot of what's seen in Monitoring Activation](/images/integrations/blackfire/blackfire-monitoring-activation.png "0.40")
+
+For more information on Blackfire monitoring features,
+see the [Blackfire documentation](https://blackfire.io/docs/monitoring-cookbooks/index).
+
+## Blackfire Profiling
+
+While your code is running, the Blackfire profiler collects deep performance metrics
+and provides full details and context of your code's behavior.
+This helps you find the root cause of performance bottlenecks.
+
+Blackfire lets you profile your application anywhere it's deployed,
+including on your local development machines.
+Using a browser extension or CLI command,
+you can profile HTTP requests, CLI scripts, Consumers, and Daemons.
+
+While HTTP requests can be profiled out-of-the-box, CLI profiling requires a
+[specific configuration](https://blackfire.io/docs/integrations/paas/upsun#cli-profiling).
+
+For more information on Blackfire profiling features,
+see the [Blackfire documentation](https://blackfire.io/docs/profiling-cookbooks/index).
+
+## Test the performance of each new deployment
+
+Blackfire's native integration with {{< vendor/name >}} enables you to test your app's performance
+every time you deploy a branch in production, staging, or development.
+Follow these steps:
+
+1.  Set up the [Blackfire Builds integration](https://blackfire.io/docs/integrations/paas/upsun#builds).
```

Comment:
https://github.com/blackfireio/blackfire.io/pull/30484/files#diff-ba99beec3e0f0f3b87f2c220a3496cf2e2ed547f6c7fce392524aecfeb2d5cb0R38

Reply:
Yes, that link